spotify/models/user.py

```diff
@@ -144,22 +109,13 @@ def __init__(
         else:
             self.library = Library(client, self)
 
-        self.token_info: TokenInfo = token_info or TokenInfo()
+        self.token_info = token_info
 
         self._cache_path: Optional[Path] = kwargs.pop("cache_path", None)
```

Comment:
Caching metadata such as a cache filepath should be associated with TokenInfo objects not individual User objects.